ros_cross_compile/ros_cross_compile.py

```diff
@@ -116,6 +117,15 @@ def parse_args(args: List[str]) -> argparse.Namespace:
         help='Relative path within the workspace to a file that provides colcon arguments. '
              'See "Package Selection and Build Customization" in README.md for more details.')
 
+    parser.add_argument(
+        '--skip-rosdep-collection',
+        action='store_true',
+        required=False,
+        help='Skip querying rosdep for dependencies. This is intended to save time'
+             'when running repeatedly during development, but has undefined behavior if'
+             "the workspace's dependencies have changed since the last time"
```

Comment:
Mismatching `"` quotes.

Reply:
Some previous other parts of the code do have mismatching quotes. Not sure if I should change or leave it..

Reply:
It's because of apostrophes. I think leave it.

Reply:
Oh, that may be because with `"` you don't have to skip the apostrophe. If that is the case, then let's leave it as is.